scripts/assignment_monitor/assignment_monitor.py

```diff
@@ -146,6 +147,34 @@ def is_assignment_processed(db, assignment):
     return bool(global_ids)
 
 
+def send_email(gmail_username, recipient_emails, assignment):
+    client = yagmail.SMTP(gmail_username)
+    recipient_emails = recipient_emails.replace(" ", "").split(",")
+    subject = str(assignment) + " has been completed"
+    body = """
+    *Assignment Completed*:
```

Comment:
Do the `*` actually do anything in the email? For slack they were to make it bold.

Reply:
Nope, good point